results/cmd/api/main_test.go

```diff
@@ -63,3 +63,33 @@ func setupTestDB(dbName string, t *testing.T) (*sql.DB, error) {
 	}
 	return db, nil
 }
+
+func TestGetTaskRun(t *testing.T) {
+	// Create a temporay database
+	testDB, err := setupTestDB("testdb", t)
+	if err != nil {
+		t.Fatalf("failed to create temp file for db: %v", err)
+	}
+
+	// connect to fake server and do testing
+	srv := &server{db: testDB}
```

Comment:
We may want to consider moving this into the setup func if this is going to be done for every test.

Reply:
done